src/runtime_src/core/include/xrt.h

```diff
@@ -634,7 +634,7 @@ XCL_DRIVER_DLLESPEC int xclGetBOProperties(xclDeviceHandle handle, xclBufferHand
  * This API is deprecated and will be removed in future release.
  * New clients should use xclGetBOProperties() instead
  */
-XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBufferHandle boHandle) __attribute__ ((deprecated));
+XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBufferHandle boHandle) XRT_DEPRECATED;
```

Comment:
Can we move it to the front as other use cases do in this file?

Reply:
?

Reply:
e.g.,:
XRT_DEPRECATED
XCL_DRIVER_DLLESPEC size_t xclWrite(xclDeviceHandle handle, enum xclAddressSpace space, uint64_t offset,
                                    const void *hostBuf, size_t size);